oti4me/abstract9-fashion-store
# Add home view

#### What does this PR do?

Adds view for the landing page

#### How can this be tested
- clone the repo
- cd into folder `abstract9-fashion-store`
- run composer install
- run `php artisan serve`
- visit http://127.0.0.1:8000 and see the look of the new landing page.

#### Relevant screenshot
<img width="1426" alt="Screenshot 2020-03-11 at 12 52 32 AM" src="https://user-images.githubusercontent.com/14099353/76369104-b4241680-6332-11ea-8e79-31c2497f692e.png">
